cmd/notary/integration_test.go

```diff
@@ -2125,7 +2131,7 @@ func TestClientKeyImport(t *testing.T) {
 	// if there is hardware available, root will only be on hardware, and not
```

Comment:
I think I may just leave it as is because they're pretty similar and re-setting up- even as a helper function will simply add time to the tests as discussed with @endophage